gqlalchemy/vendors/memgraph.py

```diff
@@ -35,17 +35,10 @@
 )
 from gqlalchemy.vendors.database_client import DatabaseClient
 from gqlalchemy.graph_algorithms.query_modules import QueryModule
+from gqlalchemy.memgraph_constants import MG_HOST, MG_PORT, MG_USERNAME, MG_PASSWORD, MG_ENCRYPTED, MG_CLIENT_NAME, MG_LAZY
```

Comment:
You can use here same techinque I mentioned in export folder in `graph_transporter.py`